pre-push checklist.md

```diff
@@ -0,0 +1,32 @@
+# Pre-push checklist
```

Comment:
why we need this in bopis feature?

Reply:
We're removing this now, it was just a reference for us during development